.docs/contributing-book/src/documentation/project-quality/documentation/code/api.md

```diff
@@ -0,0 +1,15 @@
+# API Documentation
```

Comment:
API is a common term and the ABI is basically the same thing but specialized to blockchain afaik. In order to make it more accessible I have opted for API instead of ABI. I'm open to further comments regarding this.